src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx

```diff
@@ -1,6 +1,53 @@
-import { Text } from "@artsy/palette"
-import { FC } from "react"
+import { Shelf } from "@artsy/palette"
+import { SavesItemFragmentContainer } from "Apps/CollectorProfile/Routes/Saves2/Components/SavesItem"
+import { orderBy } from "lodash"
+import { FC, useState } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "Utils/extractNodes"
+import { CollectorProfileSaves2Route_me$data } from "__generated__/CollectorProfileSaves2Route_me.graphql"
 
-export const CollectorProfileSaves2Route: FC = () => {
-  return <Text>/collector-profile/saves2</Text>
+interface CollectorProfileSaves2RouteProps {
+  me: CollectorProfileSaves2Route_me$data
 }
+
+const CollectorProfileSaves2Route: FC<CollectorProfileSaves2RouteProps> = ({
+  me,
+}) => {
+  const [selectedIndex, setSelectedIndex] = useState(0)
+  const collections = extractNodes(me.collectionsConnection)
+  // Placing the default collection at the top of the list
+  const sortedCollections = orderBy(collections, ["default"], ["desc"])
```

Comment:
**question**: maybe we should place this logic on the backend side?

Reply:
🤔 It would be better I think if we got all the data sorted from the backend side

Reply:
For now it might be okay since users won't have many wishlists but we might want to create a ticket to optimize this later 👌

Reply:
We could add param like `default_first: true` to the `collectionsConnection`, so that default collections go first, and others follow `sort` option

Reply:
created an empty ticket to refine later: https://artsyproduct.atlassian.net/browse/FX-4582

Reply:
If we consider "Default Saves" to not be part of the normal collections response, either implicitly or via a query arg, we could query for it separately.

```graphql
{
  me {
    defaultSavesCollection: collection(id: "saved-artworks") {
      name
    }
    collectionsConnection(first: 10) {
      edges {
        node {
          name
        }
      }
    }
  }
}
```

Reply:
Yeah, I was thinking something like this too.

Since `saves` and `default` args are [already supported at the Gravity endpoint](https://github.com/artsy/gravity/blob/87acd815b88cbb19dc5839628fb3ba915fe6e005/app/api/v1/collections_endpoint.rb#L11-L12), my feeling is that we could add support for them in MP and leverage those as needed.

I've got [some WIP for MP here](https://github.com/artsy/metaphysics/compare/anandaroop/support-saves-and-default-args?expand=1), and this is what that might look like in action:

<img width="2043" alt="wip" src="https://user-images.githubusercontent.com/140521/216465647-bc28963b-feeb-403b-8f8a-de0ff2c68fd9.png">